### PR TITLE
feat: add calculator, es leaf, hamburger, and lightbulb icons

### PR DESCRIPTION
**Pull request by ega713**
### 🔗 Linked issue

- CORE-4532: https://energysage.atlassian.net/browse/CORE-4532

### ❓ Type of change

- [ ] 📖 Documentation (updates to the documentation or readme)
- [ ] 🐞 Bug fix (a non-breaking change that fixes an issue)
- [ ] 👌 Enhancement (improving an existing functionality like performance)
- [x] ✨ New feature (a non-breaking change that adds functionality)
- [ ] ⚠️ Breaking change (fix or feature that would cause existing functionality to change)

### 📚 Description

- Added `icon-calculator.vue`, `icon-es-leaf.vue`, `icon-hamburger.vue` and `icon-lightbulb.vue` to `es-vue-base/src/lib-icons/`.
- These icons will be used in the mobile version of the global nav: find mockups linked from [this Confluence doc](https://energysage.atlassian.net/wiki/spaces/CPod/pages/249266400/Laying+the+Engineering+Foundation+for+the+Global+Navigation+Redesign#Mobile-Mockups-%5BinlineExtension%5D)

### 🥼 Testing

- Tested atoms/icons page in Chrome, Firefox, and Safari

### 📝 Checklist

- [x] I have linked an issue or discussion.
- [x] I have updated the documentation accordingly.
- [x] I have documented testing approach

### CODE REVIEW

File: es-vue-base/src/lib-icons/icon-lightbulb.vue [hunk 1]
```diff
@@ -0,0 +1,35 @@
...
+        fill="currentColor">
+        <path d="M19.6413 8.0574c-.2908-3.57905-3.0869-6.52059-6.6324-6.99034-2.2369-.301982-4.47385.38028-6.14034 1.84545C5.19087 4.37769 4.229 6.49157 4.229 8.71729c0 2.71781 1.45399 5.26791 3.79156 6.65481.50331.302.50331.4138.50331.5816v1.0625c0 .4586.38027.8389.83884.8389.45857 0 .83889-.3803.83889-.8389v-1.0625c0-1.0961-.67112-1.633-1.33101-2.0244-1.83427-1.0849-2.97509-3.0758-2.97509-5.21201 0-1.74479.74937-3.4001 2.05796-4.55211 1.33096-1.16319 3.04224-1.6665 4.80934-1.44281 2.729.35791 4.9548 2.70666 5.1896 5.45806.2014 2.42707-1.0289 4.63037-3.1987 5.74887-.4251.2237-1.4093.727-1.4428 2.0356v1.029c0 .4585.3802.8388.8388.8388s.8388-.3803.8388-.8388v-1.0067c0-.1677 0-.2908.5369-.5704 2.7961-1.4428 4.362-4.2725 4.1047-7.38177l.0112.02237ZM14.4516 18.7499H9.45212c-.45856 0-.83884.3803-.83884.8388 0 .4586.38028.8389.83884.8389h4.99948c.4586 0 .8389-.3803.8389-.8389 0-.4585-.3803-.8388-.8389-.8388ZM14.4516 21.3223H9.45212c-.45856 0-.83884.3802-.83884.8388s.38028.8388.83884.8388h4.99948c.4586 0 .8389-.3802.8389-.8388s-.3803-.8388-.8389-.8388Z" />
+    </svg>
+</template>
+<script>
+export default {
+    name: 'IconLightbulb',
```
Comment (tomleo) [inline es-vue-base/src/lib-icons/icon-lightbulb.vue:15]: nit: isn't light bulb two words? Should this be `icon-light-bulb.vue` with the name `IconLightBulb`?

**Review by tomleo — APPROVED** [review]
👍

File: es-vue-base/src/lib-icons/icon-lightbulb.vue (hunk 1 above)
Comment (jimlind) [reply es-vue-base/src/lib-icons/icon-lightbulb.vue:15]: Went down a bit of a rabbit hole.
The modern usage is "light bulb" (with space) but some dictionaries took a while to get with the times so while "lightbulb" (no space) is generally acceptable it is not the preferred.

Comment (ega713) [reply es-vue-base/src/lib-icons/icon-lightbulb.vue:15]: In the spirit of modernization, the names are now `icon-light-bulb.vue` and `IconLightBulb` 💡

File: es-vue-base/src/lib-icons/index.js [hunk 2]
```diff
@@ -32,12 +32,15 @@ export { default as IconLockOn } from './icon-lock-on.vue';
 export { default as IconLockOff } from './icon-lock-off.vue';
 export { default as IconLocation } from './icon-location.vue';
 export { default as IconLinkOut } from './icon-link-out.vue';
+export { default as IconLightbulb } from './icon-lightbulb.vue';
```
Comment (ericdouglaspratt) [inline es-vue-base/src/lib-icons/index.js:35]: This should probably be updated to the new `icon-light-bulb.vue` file path.

File: es-vue-base/src/lib-icons/icon-lightbulb.vue (hunk 1 above)
Comment (ericdouglaspratt) [reply es-vue-base/src/lib-icons/icon-lightbulb.vue:15]: This `icon-lightbulb.vue` should probably be deleted now that we have the new one.

**Review by ericdouglaspratt — CHANGES_REQUESTED** [review]
Ran it locally and the light bulb icon isn't showing up on the icon docs page, because the old `icon-lightbulb.vue` is still being exported from `es-vue-base` in the `index.js` and the `ds-icon-list.vue` is expecting the new name so it doesn't render. Plus I think we need to delete the old `icon-lightbulb.vue` now that we have the new one.

Otherwise looks good!

**Comment by ega713** [comment]
> Ran it locally and the light bulb icon isn't showing up on the icon docs page, because the old `icon-lightbulb.vue` is still being exported from `es-vue-base` in the `index.js` and the `ds-icon-list.vue` is expecting the new name so it doesn't render. Plus I think we need to delete the old `icon-lightbulb.vue` now that we have the new one.

I somehow ended up committing only half of the changes I made. Maybe I was committing from `es-design-system`? Thanks for the catch!

**Review by ericdouglaspratt — APPROVED** [review]
Nice - looks good now!

**Pull request merged**